tests/datadriven/test_variants.py

```diff
@@ -91,4 +98,41 @@ def testSearchAllVariants(self):
 
     def testVariantSetMetadata(self):
         # TODO get the metadata and test it.
-        pass
+        print("Testing variantset metadata " + self._vcfFileName)
+        keyMap = {}
+        for metadata in self._gaObject.getMetadata():
+            keyMap[metadata.key] = metadata
+
+        # Test version
+        metadata = keyMap["version"]
+        self.assertEqual(metadata.value, self._vcfVersion)
+
+        # Test info
+        for info_i in self._infos.keys():
```

Comment:
`key` or `infoKey` would seem to be a better variable here. We don't get much information from 'i' unless it's obviously an index variable.